example/spec/testAPI/oneTimeRule.js

```diff
@@ -144,4 +144,15 @@ describe('When I create a one-time rule via the Cumulus API', () => {
       expect(ruleCopy).toEqual(helloWorldRule);
     });
   });
+
+  it('the rule can be updated', async () => {
```

Comment:
I think this is missing this AC: `After editing an existing rule using the API, running ingest follows the updated configuration`. I think the rule should trigger a workflow again and we should check that the workflow got the updates to the rule